File: MultiplicationTable.py
def multiplicationTable(n):
    # In bảng cửu chương của số n
    for i in range(1, 11):
        print(f"{n} x {i} = {n * i}")

# Nhập số từ người dùng
n = int(input("Hãy nhập một số tự nhiên bất kỳ: "))

print(f"\nBảng cửu chương của {n}:\n")
multiplicationTable(n)

File: MultiplicationTable.c
#include <stdio.h>

void multiplicationTable(int n){
    for(int i = 1; i <= 10;i++){
        printf("%d x %d = %d\n", n, i, n*i);
    }
}

int main(){
    int n;
    printf("Hãy nhập một số tự nhiên bất kì muốn tạo bảng cửu chương: ");
    scanf("%d", &n);

    printf("Bảng cửu chương %d: \n", n);
    multiplicationTable(n);
    
    return 0; 
}